include/secp256k1.h

```diff
@@ -859,9 +859,9 @@ SECP256K1_API SECP256K1_WARN_UNUSED_RESULT int secp256k1_xonly_pubkey_tweak_add(
 
 /** Tests that output_pubkey and is_negated is the result of calling
```

Comment:
s/is the result/are the results
well, maybe better say

"Tests whether a pair (output_pubkey, is_negated) is the result of calling" or similar. I think that's more readable even though there are no actual pairs in C.

Reply:
fixed